app/views/layouts/application.html.erb

```diff
@@ -29,5 +29,7 @@
   </div>
 </div>
 
+<%= javascript_include_tag "application.js" %>
```

Comment:
I've seen other modules init instances inside itself (questionable maybe?) is this something we want to change here or at a wider scale?